app/views/core/CreateAccountModal/teacher/NcesSearchInput.coffee

```diff
@@ -66,6 +69,7 @@ NcesSearchInput = Vue.extend
     initialValue: (@value) ->
 
   mounted: ->
-    @$refs.focus.focus()
+    # TODO: probably a better Vue-ish way to specify internal nces-search-input focus behavior than props.setFocus
+    $(this.$el).find('.form-control').focus() if @setFocus is 'yes'
```

Comment:
Why not use `@$refs.focus.focus()` the same way it was handled previously?

Reply:
Move `@$refs.focus.focus()` and the `mounted` function to the parent component, SchoolInfoPanel, and then set the first element to be that reference.